content/pages/maintainers.md

```diff
@@ -0,0 +1,45 @@
+Title: Maintainer guidelines
+Tags: guideline, maintainer
+
+## Volunteering
+All [@py-pdf](https://github.com/py-pdf) members are volunteers.
+They dedicate some of their time to maintain open-source projects, answer questions and review Pull Requests.
+
+[@py-pdf](https://github.com/py-pdf) members should never be required to operate within deadlines, or even respond within a given time frame.
+
+If you are a user of a [@py-pdf](https://github.com/py-pdf) project and want something done,
+whether it is a bugfix or a feature request, your best options for achieving what you want are:
+
+* being polite and patient
+* volunteer to contribute yourself
+
+To all [@py-pdf](https://github.com/py-pdf) members, remember: [it's okay to hit pause](https://opensource.guide/best-practices/#its-okay-to-hit-pause), and take time away from volunteer open-source work.
+
+
+## Governance
+**Redaction in progress...**
+
+It's OK to [say no](https://opensource.guide/best-practices/#learning-to-say-no)
+
+<!--
+As recommended [by the opensource.guide](https://opensource.guide/leadership-and-governance/), a `GOVERNANCE.md` file in every repository could point to this page as reference.
```

Comment:
I like that. I think the governance part here should mainly define how the py-pdf organization handles to projects / people it includes + which projects get added (or when projects might get removed).

Reply:
By the way, here is the [governance document of the pypdf package](https://pypdf.readthedocs.io/en/latest/meta/project-governance.html)